examples/model_config.toml

```diff
@@ -954,20 +954,17 @@
     fare_matrix_path = "inputs\\trn\\fareMatrix.txt"
     # max expected transfer distance for mode-to-mode transfer fare table generation
     fare_max_transfer_distance_miles = 3
-    # for TAZ instead of TAPs
-    override_connectors = false # don't run prepare connector
-    override_connector_times = false # has removed all drive connectors

```

Comment:
Deleting them here is fine. But we will need to keep these and their related functions in the mtc version.